pkg/run/install/install_dashboard.go

```diff
@@ -284,20 +285,31 @@ func makeHelmRelease(log logger.Logger, name string, namespace string, username
 
 // makeValues creates a values object for installing the GitOps Dashboard.
 func makeValues(username string, passwordHash string) ([]byte, error) {
-	valuesMap := map[string]interface{}{
-		"adminUser": map[string]interface{}{
-			"create":       true,
-			"username":     username,
-			"passwordHash": passwordHash,
-		},
+	valuesMap := make(map[string]interface{})
+	if username != "" && passwordHash != "" {
+		valuesMap["adminUser"] =
+			map[string]interface{}{
+				"create":       true,
+				"username":     username,
+				"passwordHash": passwordHash,
+			}
 	}
 
-	jsonRaw, err := json.Marshal(valuesMap)
-	if err != nil {
-		return nil, fmt.Errorf("encoding values failed: %w", err)
+	analytics, _ := config.GetConfig(nil, false)
```

Comment:
💡 Would it be better to call this var `cfg` or smth. like that instead of `analytics`? We  are getting the whole CLI config here, not just the analytics field from it (even though what it contains now is related to analytics).

❔ Why are we ignoring the error here? Does it mean that the value of `valuesMap["WEAVE_GITOPS_FEATURE_TELEMETRY"]` should be always unset if we get an error when getting the config and we are not interested why exactly the error occured?